### PR TITLE
StageAda fetch 9

### PR DESCRIPTION
**Pull request by adakeita**
fetching 9 posts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,7 @@ Currently, two official plugins are available:
 
 - [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
 - [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+
+
+goodluck
```
Comment (S3ak) [inline README.md:13]: Remove

File: src/components/posts/fetchpost.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                throw new Error(`HTTP error! status: ${response.status}`);
+            }
+
+            const data = await response.json();
+
+            // Sort posts by created date
+            const sortedPosts = data.sort((a, b) => new Date(b.created) - new Date(a.created));
+
+            // Filter out posts with empty body
+            const nonEmptyPosts = sortedPosts.filter(post => post.body.trim() !== '');
```
Comment (S3ak) [inline src/components/posts/fetchpost.jsx:30]: Good thinking!

File: src/lib/api.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
 import { API_URL } from "./constants";
 
-const apiKey =
+export const apiKey =
   "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MTI3NywibmFtZSI6ImFhYWFfdXNlcm5hbWUiLCJlbWFpbCI6ImFsZXguYWxleEBzdHVkLm5vcm9mZi5ubyIsImF2YXRhciI6bnVsbCwiYmFubmVyIjpudWxsLCJpYXQiOjE2OTY5MzgzOTR9.F6R7yeEXwnD2BoP3QlcUTfvZZkJ93_Ies0APr9Oos9w";
```
Comment (S3ak) [inline src/lib/api.js:4]: Env vars should not be hardcoded, instead it should come from the .env.production / .env.local files.

Comment (alexvedt) [reply src/lib/api.js:4]: Yes thanks. Only for the purpose of testing.

**Pull request merged**